reports/Oregon.md

```diff
@@ -98,6 +98,7 @@ id: or-portland-22
 **Links**
 
 * https://twitter.com/greg_doucette/status/1268190300209586177
+* https://www.pscp.tv/IwriteOK/1OyKAYjnPgrKb?t=52m23s
```

Comment:
I know the link has the time, but can you add it as part of the hyperlink description

Reply:
Certainly!